ia-guide/ia-static.md

```diff
@@ -0,0 +1,341 @@
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>Task 6</td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>Task 7</td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>Task 8</td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>Task 9</td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+</table>
+
+Now you just need to fill in the blank spaces.
+Consider how likely the reader is to use the documentation to complete each task.
+Score a **1** for a low likelihood, **2** for a medium likelihood, and **3** for a high likelihood.
+
+**Curse of Knowledge alert!!**
+> Remember that this isn't how likely the reader is to do the task.
+> It's how likely the reader is to *use the documentation* to do the task.
+> A really experienced reader is not likely to need to the documentation for tasks like installation or setup, but they might need it for more advanced tasks.
+
+When you have the numbers in, add up each row.
+The rows that score the highest are what writers like to call *critical paths*.
+The items with the highest critical path score is the content that is most important for you to write.
+
+## Identify the mess
+
+It's time to turn our attention to what you already have.
+If you have already made a start on the docs, that's great!
+But even if you think you're starting from scratch, there's a reasonable chance you actually already have something.
+
+Have a think about it:
+
+* Have you written notes while you've been working?
+    Even if they're just scratches on a notebook on your desk, that's documentation!
+* Have you written any comments in your code?
+    You really should have! They're documentation, too.
+* Have you put a README in your code repository?
+    That's also documentation!
+* Have you given any talks or presentations about your project?
+    Even if there isn't a recording, any notes or slides you made is documentation too!
+
+Get all of these things together and look at them with your critical paths in mind.
+Do you have content here that fits all those critical paths?
+
+You will probably start to notice the gaps pretty quickly.
+It could be that you have a whole lot of content that describes how to do it, but not a lot that explains what it is.
+Or, you could have a lot of stuff aimed at beginners, and not a lot that works for experts.
+Sometimes, you might find that you have a lot of descriptions of things, but not a lot of step-by-step instructions.
+
+## What's missing?
+
+Have a think about each of your critical paths, and identify what content you need to fulfil each one.
+
+At this stage, it's normal to be feeling a little overwhelmed.
+Now you can work out what you actually need, so you can cut some things from the list.
+
+## State your intent
```

Comment:
As I was reading this document I felt this idea of setting the `objective` or `expectation` -- what is it I want my audience after reading this document to walk-away understanding.  And I think this idea of __State Your Intent__ captures it.  I think it may belong higher up in the narrative.